root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -984,9 +984,9 @@ MB.Control.autocomplete_formatters = {
             }
         };
 
-        if (item.related_entities) {
-            entityRenderer(l('Performers'), item.related_entities.performers);
-            entityRenderer(l('Location'), item.related_entities.places);
+        if (item.relatedEntities) {
+            entityRenderer(l('Performers'), item.relatedEntities.performers);
+            entityRenderer(l('Location'), item.relatedEntities.places);
```

Comment:
This should be reverted, since `related_entities` is the name of the property on the object, which comes from the server.